dipy/workflows/io.py

```diff
@@ -257,3 +258,39 @@ def run(self, input_files, vol_idx=0, out_dir='',
             save_nifti(osplit, split_vol, affine, image.header)
 
             logging.info('Split volume saved as {0}'.format(osplit))
+
+
+class ConvertDipyMrtrixFlow(Workflow):
```

Comment:
Please use a more meaningful name for the workflow: what does it convert ? e.g.  `ConvertDipyMrtrixSHFlow`

Reply:
I agree, maybe just `ConvertSHFlow`